src/com/amazon/ion/IonWriter.java

```diff
@@ -113,7 +113,9 @@
      * In particular, when writing binary Ion data, Ion's length-prefixed
      * encoding requires a complete top-level value to be written at once.
      * <p>
-     * This feature can be used to flush buffered data before writing more
+     * If localSymbolTableAppend is enabled (see
+     * {@link com.amazon.ion.system.IonBinaryWriterBuilder#withLocalSymbolTableAppendEnabled}),
+     * this feature can be used to flush buffered data before writing more
```

Comment:
In the documentation for `finish()` (below), please update the following sentence

> Applications that produce long streams with a limited number of unique symbols should use {@link #flush()} instead to avoid re-declaring the local symbol table unnecessarily.

to something like

> Applications that produce long streams with a limited number of unique symbols should enable localSymbolTableAppend (see {@link com.amazon.ion.system.IonBinaryWriterBuilder#withLocalSymbolTableAppendEnabled}) and {@link #flush()} instead to avoid re-declaring the local symbol table unnecessarily.

Reply:
done